### PR TITLE
Improve the 'Setting up Cluster Mesh' page.

### PR DESCRIPTION
**Pull request by bmcustodio**
Please ensure your pull request adheres to the following guidelines:

- [x] For first time contributors, read [Submitting a pull request](http://docs.cilium.io/en/stable/contributing/contributing/#submitting-a-pull-request)
- [x] All code is covered by unit and/or runtime tests where feasible.
- [x] All commits contain a well written commit description including a title,
      description and a `Fixes: #XXX` line if the commit addresses a particular
      GitHub issue. 
- [x] All commits are signed off. See the section [Developer’s Certificate of Origin](http://docs.cilium.io/en/stable/contributing/contributing/#dev-coo).
- [x] Provide a title or release-note blurb suitable for the release notes.
- [x] Thanks for contributing!

This PR updates the _Setting up Cluster Mesh_ page in order to suggest that services of type `NodePort` are also supported, following https://github.com/cilium/clustermesh-tools/pull/1.

Fixes: N/A

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/9725)

### CODE REVIEW

**Comment by joestringer** [comment]
test-docs-please

**Review by aanm — APPROVED** [review]

File: Documentation/gettingstarted/clustermesh.rst [hunk 1]
```diff
@@ -68,12 +68,20 @@ argument or ``cluster-id`` ConfigMap option. The value must be between 1 and
 
 Repeat this step for each cluster.
 
+.. note::
+
+   This can also be done by passing `--set global.cluster.id=<id>` and
```
Comment (aanm) [inline Documentation/gettingstarted/clustermesh.rst]: needs to be double \` between `--set global.cluster.id=<id>`

File: Documentation/gettingstarted/clustermesh.rst [hunk 2]
```diff
@@ -68,12 +68,20 @@ argument or ``cluster-id`` ConfigMap option. The value must be between 1 and
...
+   `--set global.cluster.name=<name>` to `helm template` when installing or
```
Comment (aanm) [inline Documentation/gettingstarted/clustermesh.rst]: needs to be double \` between `--set global.cluster.name=<name>` and `helm template`

**Review by aanm — CHANGES_REQUESTED** [review]

**Review by joestringer — COMMENTED** [review]
Thanks for the contribution!

It looks like the `make test-docs` target is failing, most likely due to the areas that @aanm commented on. Otherwise LGTM. You can feel free to ignore the `maintainers-little-helper` failures below, we're working through some issues in the github actions we use.

**Comment by bmcstdio** [comment]
@aanm @joestringer thank you for your reviews, I've updated the branch accordingly. :+1:

**Comment by aanm** [comment]
test-docs-please

**Review by joestringer — APPROVED** [review]

**Comment by bmcstdio** [comment]
@aanm @joestringer thank you for your feedback! I don't know how the publishing process for documentation works, but I believe that https://github.com/cilium/clustermesh-tools/pull/1 should be merged before this gets published. 🙂

**Pull request merged**